modules/dnn/test/test_onnx_conformance_layer_filter_opencv_all_denylist.inl.hpp

```diff
@@ -0,0 +1,30 @@
+"test_add_bcast",
+"test_averagepool_2d_pads_count_include_pad",
+"test_averagepool_2d_precomputed_pads_count_include_pad",
+"test_averagepool_2d_same_lower",
+"test_cast_FLOAT_to_STRING",
+"test_cast_STRING_to_FLOAT",
+"test_castlike_FLOAT_to_STRING_expanded",
+"test_castlike_STRING_to_FLOAT_expanded",
+"test_clip",
+"test_clip_default_max",
+"test_clip_default_min",
+"test_clip_inbounds",
```

Comment:
> test_clip_inbounds

OpenCL FP32 passes this test.
https://github.com/rogday/opencv/blob/0aba6f51d4619570792c72aa700cdcfa8a992cf2/modules/dnn/test/test_onnx_conformance_layer_filter__opencv.inl.hpp#L335-L338

Reply:
The model from this test is broken.

Reply:
This model has 3 inputs:
- X
- min (input, not a parameter)
- max (input, not a parameter)

Reply:
Yes. I believe min and max inputs should be constant blobs in our terms.
https://github.com/onnx/onnx/blob/master/docs/Operators.md#Clip

Reply:
Perhaps we should add check in parser code (preferable in this PR). And add this testcase into `parser_denylist`.

BTW, I have updated a bit opencv_extra PR (to emit denylist for parser part).